src/en/components/button/use-case.md

```diff
@@ -55,65 +56,76 @@ Use the submit type when you're expecting a person to:
 
 - Send a form or a request.
 - Save information.
-- Sign in or sign out of a secure space.
+- Sign in or sign out of an account.
 
 ### Reset type
 
 Use the reset type when you're expecting a person to:
 
 - Clear data entered on a form.
 
+Tip: Use the reset type sparingly as it’s a destructive action.
+
 ### Link type
 
-Use the link type when you're expecting a person to:
+Use the link type when you’re expecting a person to:
 
-- Link to other pages or external sites, using the link type.
+- Navigate to other pages or external sites.
 
-Note: Always use the [link component]({{ links.link }}) when you need to embed a link in body or paragraph text.
+Note: Use the [link component]({{ links.link }}) when you need to embed a link in paragraph text or navigation bars.
 
 ## Choose a button role
 
-A role is a button sub-type that has a specific use on a page.
+The button role changes the styling of a button to visually signal its action. Pick the role that best fits the action.
 
 <div class="remove-empty-p">
 <gcds-grid columns="1fr" columns-tablet="1fr 2fr" align-items="start">
-  {% componentPreview "Primary button preview" "px-225 py-300" "" %}
+  {% componentPreview "Primary button preview" "px-225 py-300" "mt-500" %}
   <gcds-button button-role="primary">Submit</gcds-button>
   {% endcomponentPreview %}
   <div>
-    <h3>Primary button uses</h3>
+    <h3>Primary button</h3>
     <ul class="list-disc mb-300">
-      <li>The most important action on a page.</li>
       <li>Critical actions in a flow or as the default button.</li>
       <li>Only use a single time.</li>
     </ul>
-    <p>Note: More than one main call to action on the same page reduces impact and makes it harder for a person to know what to do next.</p>
+    <p>Note: More than one main call to action on the same page makes it harder for a person to know what to do next.</p>
   </div>
-</gcds-grid>
-<br/>
-<gcds-grid columns="1fr" columns-tablet="1fr 2fr" align-items="start">
-  {% componentPreview "Secondary button preview" "px-225 py-300" "" %}
+
+  {% componentPreview "Start button preview" "px-225 py-300" "mt-500" %}
+  <gcds-button button-role="start">Start</gcds-button>
+  {% endcomponentPreview %}
+  <div>
+    <h3>Start button</h3>
+    <p><i>Formerly called supertask button.</i></p>
```

Comment:
Can we switch this back to an `<em>`? The `<i>` tag is not accessible because it only visually emphasizes text.

Reply:
Reading through the comments and if it's just for searching and not meant as an emphasis indicator, we should be able to remove the tag altogether. If a user searches for "super task", they will find it either way.